framework/src/st_things/things_stack/things_api.h

```diff
@@ -27,38 +27,257 @@
 extern "C" {
 #endif							// __cplusplus
 
+//--------------------------------------------------------------------------------
+/**
+ * @brief callback for handling result of reset.
+ * @param is success result of reset.(1 : success, = : failure)
+ */
 typedef void (*things_reset_result_func_type)(int result);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief callback for informing the st thing to close the PIN display.
+ */
 typedef void (*things_pin_display_close_func_type)(void);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief callback for carrying the randomly generated PIN info.
+ * @param pin_data : PIN data in string format.
+ * @param pin_size : Length of the PIN string.
+ */
 typedef void (*things_pin_generated_func_type)(char *pin_data, size_t pin_size);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief callback function for getting the easysetup states.
+ * @param state : hold the enrollee state.
+ */
 typedef void (*things_get_easysetup_state_func_type)(things_es_enrollee_state_e state);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief callback for connection result of cloud.It is called when finish Cloud-Connection.
+ * @param result : 1 = published resource to cloud , 0 =failed to publish resource
```

Comment:
please add white space between "=" and "failed"

Reply:
done.